### PR TITLE
fix(env): convert all env vars to SCREAMING_CASE from camelCase

### PR DESCRIPTION
**Pull request by hiroyukikumazawa**

### CODE REVIEW

**Comment by hiroyukikumazawa** [comment]
refs: #310

**Comment by ygrishajev** [comment]
hey @hiroyukikumazawa looks like a bunch of envs are not updated here https://github.com/akash-network/console/blob/main/packages/database/chainDefinitions.ts
Would be real nice if you kindly could check if the apps are at least starting without errors.

Note: I found it's only `ActiveChain` and it only affects indexer. @Redm4x @baktun14 are other used at all?

**Comment by hiroyukikumazawa** [comment]
> hey @hiroyukikumazawa looks like a bunch of envs are not updated here https://github.com/akash-network/console/blob/main/packages/database/chainDefinitions.ts Would be real nice if you kindly could check if the apps are at least starting without errors.
> 
> Note: I found it's only `ActiveChain` and it only affects indexer. @Redm4x @baktun14 are other used at all?

Ok, I see
And the apps projects are working correctly with this updates.
What about your side? @ygrishajev

**Comment by ygrishajev** [comment]
> > hey @hiroyukikumazawa looks like a bunch of envs are not updated here https://github.com/akash-network/console/blob/main/packages/database/chainDefinitions.ts Would be real nice if you kindly could check if the apps are at least starting without errors.
> > Note: I found it's only `ActiveChain` and it only affects indexer. @Redm4x @baktun14 are other used at all?
> 
> Ok, I see And the apps projects are working correctly with this updates. What about your side? @ygrishajev

@hiroyukikumazawa yea, that should do, thanks! I'd still like @Redm4x or @baktun14 to check this out too

**Comment by baktun14** [comment]
> hey @hiroyukikumazawa looks like a bunch of envs are not updated here https://github.com/akash-network/console/blob/main/packages/database/chainDefinitions.ts Would be real nice if you kindly could check if the apps are at least starting without errors.
> 
> Note: I found it's only `ActiveChain` and it only affects indexer. @Redm4x @baktun14 are other used at all?

We only need akash related env var in this file. Others are to run the indexer for other chains, which we don't care for console.

**Review by baktun14 — APPROVED** [review]

**Review by Redm4x — APPROVED** [review]

**Comment by ygrishajev** [comment]
Hey @hiroyukikumazawa pls rebase and I'll deploy this to staging so everyone can test. Then we'll merge

**Comment by ygrishajev** [comment]
@hiroyukikumazawa any chance you could rebase soon?

**Comment by hiroyukikumazawa** [comment]
> We only need akash related env var in this file. Others are to run the indexer for other chains, which we don't care for console

I will rebase now

**Comment by ygrishajev** [comment]
@hiroyukikumazawa  I've done relevant changes in deploy-web last week, which change the location of configs and also renames vars to uppercase. Please accept 'main' changes in case of conflicts.

Ref https://github.com/akash-network/console/pull/355

**Comment by hiroyukikumazawa** [comment]
> @hiroyukikumazawa I've done relevant changes in deploy-web last week, which change the location of configs and also renames vars to uppercase. Please accept 'main' changes in case of conflicts.

ok

**Comment by hiroyukikumazawa** [comment]
@ygrishajev please review again

**Comment by ygrishajev** [comment]
@hiroyukikumazawa I see another conflict, would you handle that?

**Comment by hiroyukikumazawa** [comment]
@ygrishajev please check again

**Comment by ygrishajev** [comment]
@hiroyukikumazawa thanks a bunch! So I've deployed api to our beta and have indexer running locally. I'd like @Redm4x and @baktun14 to verify it once again and merge this.

I'm going to squash commits on merge but you're welcome to do so yourself in the branch if you like so.


Locally I have these vars set:
```
ACTIVE_CHAIN=akashSandbox
AKASH_DATABASE_CS=postgres://postgres:password@db:5432/console-akash-sandbox
ActiveChain=akashSandbox
``` 
image

**Comment by hiroyukikumazawa** [comment]
> ActiveChain

@ygrishajev  was `ActiveChain=akashSandbox` needed in indexer?

**Comment by ygrishajev** [comment]
> > ActiveChain
> 
> @ygrishajev was `ActiveChain=akashSandbox` needed in indexer?

oh, indeed I found a bunch of them used
image

**Comment by hiroyukikumazawa** [comment]
@ygrishajev 
I thought `activeChain` is not a env var. 
Instead, `ActiveChain` is in the readme.
And in my local, without ActiveChain env var it's working.

**Comment by hiroyukikumazawa** [comment]
@ygrishajev done squash commit

**Comment by ygrishajev** [comment]
> @ygrishajev I thought `activeChain` is not a env var. Instead, `ActiveChain` is in the readme. And in my local, without ActiveChain env var it's working.

@hiroyukikumazawa https://github.com/akash-network/console/pull/335/files#diff-a7b12b4516cdcf3f8b1898a997e54d3b4a328a14bb80aecf30e2e23734e3a3a8R412

**Comment by hiroyukikumazawa** [comment]
> > @ygrishajev I thought `activeChain` is not a env var. Instead, `ActiveChain` is in the readme. And in my local, without ActiveChain env var it's working.
> 
> @hiroyukikumazawa https://github.com/akash-network/console/pull/335/files#diff-a7b12b4516cdcf3f8b1898a997e54d3b4a328a14bb80aecf30e2e23734e3a3a8R412

wdym?

File: apps/api/src/utils/env.ts [hunk 1]
```diff
@@ -4,20 +4,20 @@ export const env = z
...
-    SecretToken: z.string().optional(),
+    REST_API_NODE_URL: z.string().optional(),
+    SERVER_ORIGIN: z.string().optional().default("http://localhost:3080"),
+    AKASHLYTICS_GITHUB_PAT: z.string().optional(),
+    AUTH0_JWKS_URI: z.string().optional(),
+    AUTH0_AUDIENCE: z.string().optional(),
+    AUTH0_ISSUER: z.string().optional(),
+    WEBSITE_URL: z.string().optional(),
+    SECRET_TOKEN: z.string().optional(),
     ProviderUptimeGracePeriodMinutes: z
```
Comment (Redm4x) [inline apps/api/src/utils/env.ts]: `ProviderUptimeGracePeriodMinutes` was missed

**Review by Redm4x — CHANGES_REQUESTED** [review]

File: apps/api/src/utils/env.ts (hunk 1 above)
Comment (hiroyukikumazawa) [reply apps/api/src/utils/env.ts]: ok, will update now

Comment (hiroyukikumazawa) [reply apps/api/src/utils/env.ts]: please check again @Redm4x

**Review by Redm4x — APPROVED** [review]

**Review by ygrishajev — APPROVED** [review]

**Pull request merged**